dev-scheme/racket/racket-8.2-r1.ebuild

```diff
@@ -19,12 +19,14 @@ LICENSE="
 	chez? ( Apache-2.0 )
 	!chez? ( LGPL-3 )
 "
-# Bytecode generated by Racket is not compatible between versions
+# Bytecode generated by Racket is not compatible between versions.
+# In this case PV == bytecode version,
+# but for _patch, _pre, etc. it has to be set manually.
```

Comment:
What does this mean? Why not just use ${PF} or similar?

Reply:
What I meant to say is that here `PV == SLOT`.
The SLOT should point to the bytecode version.
So in this case `PV == bytecode version` is also true.

Reply:
Oh, I think I understand you now. Maybe clarify the comment a bit if you can?